slidedeck/src/slides/07-context/00-context.pug

```diff
@@ -0,0 +1,127 @@
+section
+  h1 Context
+
+section
+  h3 What is Context?
+  p A pure-React mechanism to share data between multiple components
+  p Can be used to replace many use cases of Redux
+
+section
+  h3 What makes up a Context?
+  p A context is composed of three elements
+  ul
+    li The Context object
+    li A Provider that establishes bounds for the Context
+    li One or more Consumers within those bounds that use the Context
+
+section
+  h2 How do I use Context?
+
+section
+  h2 Step One
+  h3 Create a Context
+
+  script(type="text/plain", class="language-jsx").
+    export const WeatherContext = React.createContext({
+      current: 'sunny',
+      tomorrow: 'rainy'
+    });
+
+section
+  h2 Step Two
+  h3 Create a Provider
+
+  script(type="text/plain", class="language-jsx").
+    class WeatherController extends React.Component {
+      constructor() {
+        this.state = {
```

Comment:
Do you think that it would it be worth the effort to change our examples to define state as a class property? I like the simplicity of not having a `constructor`

Reply:
I'd be fine switching over. Maybe a follow-on PR to tackle that across all the slides?

Reply:
Yeah, agreed